apps/condo/domains/common/components/Alert.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import { Alert as DefaultAlert, AlertProps } from 'antd'
+import React, { ComponentProps } from 'react'
+
+import { colors, fontSizes } from '@condo/domains/common/constants/style'
+
+const getAlertColorsByType = (type: ComponentProps<typeof Alert>['type']) => {
+    const defaultColors = {
+        bg: colors.infoAlertBg,
+        message: colors.infoAlertMessage,
+        description: colors.infoAlertDescription,
+    }
+
+    switch (type) {
+        case 'info': return defaultColors
+
+        default: return defaultColors
+    }
+}
+
+const StyledAlert = styled(DefaultAlert)<{ bgColor, messageColor, descriptionColor }>`
```

Comment:
Looks too strange for me. I guess it' better not to invent a bicycle here and use default ant notification with TODO to replace it.

You can use `DOMA-3753` as a parent task to store UI todos